nilearn/datasets/atlas.py

```diff
@@ -863,3 +863,77 @@ def fetch_atlas_allen_2011(data_dir=None, url=None, resume=True, verbose=1):
     params.extend(list(zip(keys, sub_files)))
 
     return Bunch(**dict(params))
+
+
+def fetch_atlas_surf_destrieux(data_dir=None, url=None,
+                               resume=True, verbose=1):
+    """Download and load Destrieux et al, 2010 cortical atlas.
+
+    This atlas returns 74 sulco-gyral structures per hemisphere distributed
```

Comment:
One label is "Unknown" and one is "Medial wall", so only 74 real labels. Still seems kind of confusing, I'll fix it